main.py

```diff
@@ -95,6 +97,10 @@
     (API_BASE + '/features/<int:feature_id>/approvals/<int:field_id>/comments',
      comments_api.CommentsAPI),
 
+    (API_BASE + '/fielddefs', fielddefs_api.FieldDefsAPI),
+
+    (API_BASE + '/processes/<int:feature_id>', processes_api.ProcessesAPI),
```

Comment:
If the URL is `/processes/<int>`, I would expect that that int is the feature_type.   I think it is a fine idea to use the feature_id in the request because that allows you to initiate this request without waiting for the feature itself to load.  However, if the parameter is going to be the feature_id, then the url should probably be `/feature/<int:feature_id>/process`.  That logic does not apply to fielddefs because that request has no ID in the URL.

Reply:
Changed it to `/feature/<int:feature_id>/process`. I think for now it's simpler and easier to debug to just use Promise.all to send out all requests at once.